### PR TITLE
Add faster tokenizers cpp source code

### PR DESCRIPTION
**Pull request by joey12300**
### PR types

New features
### PR changes

APIs
### Description

Add faster tokenizers cpp source code

### CODE REVIEW

File: faster_tokenizers/LICENSE [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright (c) 2016 PaddlePaddle Authors. All Rights Reserved.
```
Comment (ZeyuChen) [inline faster_tokenizers/LICENSE]: 这里请改成2022

File: faster_tokenizers/setup.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+long_description = "PaddleNLP Faster Tokenizer Library written in C++ "
+setup(
+    name="faster_tokenizers",
+    version="0.0.1",
+    author="PaddlePaddle Speech and Language Team",
+    author_email="paddlesl@baidu.com",
+    description=long_description,
+    long_description=long_description,
+    zip_safe=False,
+    url="https://github.com/PaddlePaddle/PaddleNLP",
```
Comment (ZeyuChen) [inline faster_tokenizers/setup.py]: https://github.com/PaddlePaddle/PaddleNLP/faster_tokenizers

**Review by ZeyuChen — COMMENTED** [review]
只有一些小问题，可以快速修改后快速合入一波。

File: faster_tokenizers/LICENSE (hunk 1 above)
Comment (joey12300) [reply faster_tokenizers/LICENSE]: done

File: faster_tokenizers/setup.py (hunk 2 above)
Comment (joey12300) [reply faster_tokenizers/setup.py]: done

**Review by ZeyuChen — APPROVED** [review]

**Pull request merged**